### PR TITLE
6.0 bundled theme version bump

### PR DESCRIPTION
**Pull request by peterwilsoncc**
https://core.trac.wordpress.org/ticket/55754

### CODE REVIEW

File: src/wp-content/themes/twentyeleven/readme.txt [hunk 1]
```diff
@@ -2,7 +2,7 @@
 Contributors: wordpressdotorg
 Requires at least: WordPress 3.2
 Tested up to: 5.9
```
Comment (costdev) [inline src/wp-content/themes/twentyeleven/readme.txt]: Does this need bumped to 6.0? Same question for the other themes

**Review by costdev — COMMENTED** [review]
This review was a typo check to make sure the version bumps were consistent in each theme. I left a question about the `readme.txt` "Tested up to: 5.9" versions.

File: src/wp-content/themes/twentyeleven/readme.txt (hunk 1 above)
Comment (desrosj) [reply src/wp-content/themes/twentyeleven/readme.txt]: Yes, it should be. it was adjusted in others, but seems a few were missed.

File: src/wp-content/themes/twentyfifteen/readme.txt [hunk 2]
```diff
@@ -2,7 +2,7 @@
 Contributors: wordpressdotorg
 Requires at least: WordPress 4.1
 Tested up to: 5.9
```
Comment (desrosj) [inline src/wp-content/themes/twentyfifteen/readme.txt]:
```suggestion
Tested up to: 6.0
```

File: src/wp-content/themes/twentyseventeen/readme.txt [hunk 3]
```diff
@@ -1,7 +1,7 @@
 === Twenty Seventeen ===
 Contributors: wordpressdotorg
 Tested up to: 5.9
```
Comment (desrosj) [inline src/wp-content/themes/twentyseventeen/readme.txt]:
```suggestion
Tested up to: 6.0
```

File: src/wp-content/themes/twentysixteen/readme.txt [hunk 4]
```diff
@@ -1,7 +1,7 @@
 === Twenty Sixteen ===
 Contributors: wordpressdotorg
 Tested up to: 5.9
```
Comment (desrosj) [inline src/wp-content/themes/twentysixteen/readme.txt]:
```suggestion
Tested up to: 6.0
```

File: src/wp-content/themes/twentyten/readme.txt [hunk 5]
```diff
@@ -1,7 +1,7 @@
 === Twenty Ten ===
 Contributors: wordpressdotorg
 Tested up to: 5.9
```
Comment (desrosj) [inline src/wp-content/themes/twentyten/readme.txt]:
```suggestion
Tested up to: 6.0
```

File: src/wp-content/themes/twentytwelve/readme.txt [hunk 6]
```diff
@@ -1,7 +1,7 @@
 === Twenty Twelve ===
 Contributors: wordpressdotorg
 Tested up to: 5.9
```
Comment (desrosj) [inline src/wp-content/themes/twentytwelve/readme.txt]:
```suggestion
Tested up to: 6.0
```

File: src/wp-content/themes/twentytwentyone/readme.txt [hunk 7]
```diff
@@ -3,7 +3,7 @@ Contributors: wordpressdotorg
 Requires at least: 5.3
 Tested up to: 5.9
```
Comment (desrosj) [inline src/wp-content/themes/twentytwentyone/readme.txt]:
```suggestion
Tested up to: 6.0
```

File: src/wp-content/themes/twentythirteen/functions.php [hunk 8]
```diff
@@ -319,7 +319,7 @@ function twentythirteen_scripts_styles() {
 	wp_enqueue_style( 'genericons', get_template_directory_uri() . '/genericons/genericons.css', array(), '3.0.3' );
 
 	// Loads our main stylesheet.
```
Comment (desrosj) [inline src/wp-content/themes/twentythirteen/functions.php:321]: It looks like several other styles and scripts did not have their versions bumped. I had always just assumed that the theme version was used, but seems that for older default themes, that's not the case.

@peterwilsoncc what do you think about changing all of these date versions to be accurate now? Or, we could just open a follow up ticket.

**Review by desrosj — APPROVED** [review]
Looks good to me pending a few comments/suggestions I made on the PR. Thanks for handling this!

**Comment by mehedifoysal** [comment]
I checked all of them and look perfect to me

File: src/wp-content/themes/twentyeleven/readme.txt (hunk 1 above)
Comment (peterwilsoncc) [inline src/wp-content/themes/twentyeleven/readme.txt]:
```suggestion
Tested up to: 6.0
```

**Comment by peterwilsoncc** [comment]
@desrosj @costdev I think I've caught all of the _tested up to_ bumps that are required now. Thanks for catching that.

> It looks like several other styles and scripts did not have their versions bumped. I had always just assumed that the theme version was used, but seems that for older default themes, that's not the case.
> 
> @peterwilsoncc what do you think about changing all of these date versions to be accurate now? Or, we could just open a follow up ticket.

I reviewed [the commits in the `wp-content/themes` directory](https://github.com/WordPress/wordpress-develop/commits/trunk/src/wp-content/themes) and manually bumped the assets that had changed.

I think using the theme version is a good idea but let's use a follow up ticket. For themes with build tools it might we worth using wp-scripts to generate the version as it will be more performant. But let's discuss that there.

**Comment by SergeyBiryukov** [comment]
Merged in https://core.trac.wordpress.org/changeset/53418.

**Pull request closed without merging**